frontend/src/pages/AnnouncementPage.js

```diff
@@ -0,0 +1,330 @@
+// SPDX-License-Identifier: MIT
+// SDPX-FileCopyrightText: 2021 Que Le <b.le@tu-berlin.de>
+
+import React, {
+	useEffect,
+	useState
+} from "react";
+import appwriteApi from "../api/appwriteApi";
+
+import Grid from "@mui/material/Grid";
+import { makeStyles } from "@mui/styles";
+import {
+	Button, Alert,
+	Table, TableBody,
+	TableCell, TableRow,
+	TextField, Typography,
+} from "@mui/material";
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import Collapse from "@mui/material/Collapse";
+
+function InputFields({ defaultTitle, defaultContent, titleComponenId, contentComponenId }) {
+	const useStyles = makeStyles(() => ({
+		input: {
+			color: "#FFF",
+		},
+	}));
+
+	const classes = useStyles();
+	return <Grid container spacing={2}>
+		<Grid item xs={12}>
+			<TextField
+				required
+				fullWidth
+				name="title"
+				label="Title"
+				id={titleComponenId}
+				color="warning"
+				defaultValue={defaultTitle}
+				inputProps={{ className: classes.input }}
+			/>
+		</Grid>
+		<Grid item xs={12}>
+			<TextField
+				required
+				fullWidth
+				name="content"
+				label="Content"
+				id={contentComponenId}
+				defaultValue={defaultContent}
+				inputProps={{ className: classes.input }}
+			/>
+		</Grid>
+	</Grid>;
+}
+
+function AnnouncementEntry({ announcement, editing, setEditing, userIsAdmin }) {
+	const created_at = new Date(announcement.created_at * 1000);
+	const formated_created_at =
+		created_at.getDate() + "/" +
+		created_at.getMonth() + "/" +
+		created_at.getFullYear() + " " +
+		created_at.getHours() + ":" +
+		created_at.getMinutes();
```

Comment:
There are better ways of formatting time. In particular, e,g, for time single digit minutes, this results in "16/10/2021 23:3". Please add leading zeros.